### DISCUSSION TITLE
How to determine current instance is `server` or `generate` argument inside HexoJS Plugin?

### DISCUSSION THREAD

**Discussion by dimaslanjaka** [hexojs/hexo#5225] [Question / Answer]
How to determine current instance is `server` or `generate` argument inside HexoJS Plugin Without parsing `process.argv` (which regenerate more memory usages). Is there has indicator variable from hexo context?

**Comment by dimaslanjaka** [comment]
@SukkaW help answer pls

**Comment by stevenjoezhang** [comment]
There is no such indicator variable

**Reply by dimaslanjaka** [reply]
i hope next version the CLI argument object exported through hexo context :)
